app/assets/javascripts/sitewide/tiny_mce_file_upload_plugin.js

```diff
@@ -300,7 +300,9 @@
         images = $.map($('img', iframe), e => {
           return e.dataset.mceToken;
         });
-        $('#' + editor.id).next()[0].value = JSON.stringify(images);
+        if ($('#' + editor.id).next()[0]) {
```

Comment:
Maybe we need some error handling here. Because if we don't find this `next()` element it will mean that we will not save images.

Reply:
I assume that there are no images If it doesn't find next element